### PR TITLE
Fix patching of some algorithms for using more input types

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

**Comment by OnlyDeniko** [comment]
https://ecmd.inn.intel.com/commander/link/jobDetails/jobs/eb359926-3ca9-f104-96c2-8cdcd4b723a1?

File: daal4py/sklearn/linear_model/_logistic_path_0_22.py [hunk 1]
```diff
@@ -972,7 +974,8 @@ def daal4py_predict(self, X, resultsToEvaluate):
     multinomial = (self.multi_class in ["multinomial", "warn"] or
                    self.classes_.size == 2 or resultsToEvaluate == 'computeClassLabels')
 
-    if daal_check_version(((2021,'P', 1))) and fptype is not None and not sparse.issparse(X) and multinomial:
+    if daal_check_version(((2021,'P', 1))) and fptype is not None and not sparse.issparse(X) \
+       and multinomial and isinstance(self.coef_, np.ndarray):
```
Comment (PetrovKP) [inline daal4py/sklearn/linear_model/_logistic_path_0_22.py:978]: why require an array type - `self.coef_`?

**Comment by OnlyDeniko** [comment]
https://ecmd.inn.intel.com/commander/link/jobDetails/jobs/eb36214b-d40a-f151-9d51-8cdcd4b723a1?

**Comment by OnlyDeniko** [comment]
http://intel-ci.intel.com/eb36214b-d40a-f151-9d51-8cdcd4b723a1

**Comment by OnlyDeniko** [comment]
http://intel-ci.intel.com/eb362fb5-f495-f184-b8d7-8cdcd4b723a1

**Review by PetrovKP — APPROVED** [review]

**Review by Pahandrovich — APPROVED** [review]
LGTM

**Pull request merged**